vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -23602,3 +23602,106 @@ THE UCUM TABLE (IN ALL FORMATS), UCUM DEFINITIONS, AND SPECIFICATION ARE PROVIDE
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "All Units Ontology Version 2.1.16" ;
 .
+
+unit:DEG_C_GROWING_CEREAL-DAY
+    a qudt:Unit ;
+    rdfs:label "Growing Degree Days (Cereals)" ;
+    qudt:conversionMultiplier "86400"^^xsd:double ;
+    qudt:conversionOffset "0.0"^^xsd:double ;
+    qudt:hasDimensionVector qkdv:A0E0L0I0M0H1T1D0 ;
+    qudt:plainTextDescription "The sum of excess temperature over 5.5°C, where the temperature is the mean of the minimum and maximum atmospheric temperature in a day. This measure is appropriate for most cereal crops." ;
+    qudt:hasQuantityKind quantityKind:GrowingDegreeDay_Cereal ;
+    rdfs:isDefinedBy <https://data.agrimetrics.co.uk/ontologies/qudt-extension> .
+
+unit:M2-PER-M2
+    a  qudt:Unit;
+    rdfs:label "square metre per square metre" ;
+    qudt:hasDimensionVector qkdv:A0E0L2I0M0H0T0D0 ;
+    qudt:hasQuantityKind quantityKind:AreaRatio ;
+    qudt:conversionMultiplier "1"^^xsd:double ;
+    qudt:conversionOffset "0.0"^^xsd:double ;
+    qudt:plainTextDescription "A square metre unit of area per square metre"^^rdf:HTML ;
+    <http://qudt.org/schema/qudt/symbol> "m^2/m^2" ;
+    rdfs:isDefinedBy <https://data.agrimetrics.co.uk/ontologies/qudt-extension> .
+
+unit:SHANNON_DIVERSITY
```

Comment:
If you never want users to instantiate this (abstract) unit, you can remove the qudt:hasQuantityKind triple. Then only the specific ones (LOG10, etc.) would show up as applicableUnit when examining the ShannonDiversityIndex.

Reply:
(The applicableUnit generation algorithm is documented [here](https://github.com/qudt/qudt-public-repo/wiki/Advanced-User-Guide#4-computing-applicable-units-for-a-quantitykind).)

Reply:
Hi Steve, 

I'm Monika's colleague.

The uses of this unit that I've seen usually don't specify which base they've used for the logarithm, but it does appear that multiple bases are in use -- I wasn't able to pin down any specific recommendations about which one should be used. We left the "abstract" version in so that it's possible to indicate the type of unit without having to specify a base that's undocumented by the upstream generators of the data we're trying to annotate.

If that deliberate vagueness is against the intent of QUDT, then by all means we can drop the qudt:hasQuantityKind, but it's useful to have it, I think.

Reply:
Here's what I propose, which is what we did for unit:OZ and others. The ounce is a measure of force, mass, and volume. We chose what we believed to be the "most common" one - mass - and used unit:OZ to mean that. The others are named unit:OZ_F and unit:OZ_VOL (with yet a further qualifier of _UK or _US). The point was that the unqualified version means **_something_** unambiguous, which can be interrogated.

In your case, we would collectively choose a "most common" logarithmic scale, and anoint unit:SHANNON_DIVERSITY to mean that. The other variants would have the qualifier. The description would make each one clear. You may also want to include:
rdfs:seeAlso <the other variants> 
... in each one, but that is your choice.

Reply:
@steveraysteveray Thanks. We have decided that for now, we will not submit the Shannon index units. We will make another submission on this, once we get some more clarity at our end.

Reply:
OK. Another option you might want to consider in the meantime is to add the three qualified versions which would support applications where the scale is known.